pkg/agent/runner.go

```diff
@@ -1507,6 +1518,8 @@ func (r *Runner) doVMUpdate(
 
 	// We couldn't update the VM
 	if err != nil {
+		r.global.metrics.neonvmRequestsOutbound.WithLabelValues(fmt.Sprintf("[error: %s]", util.RootError(err))).Inc()
```

Comment:
cardinality concerns.

Reply:
This is the same metric that you've already mentioned — resolving this in favor of that thread.